manifests/5.1/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -173,6 +173,18 @@ spec:
                       items:
                         type: string
                       type: array
+                    parse:
+                      default: json
+                      description: "Parse enables parsing of the log entries into structured logs \n Logs are parsed according to parse value, only `json` is supported as of now."
+                      enum:
+                      - json
+                      type: string
+                    schemaKey:
+                      description: SchemaKey specifies the metadata key to be used as name of log schema It takes precedence over SchemaName
```

Comment:
@alanconway i missed this detail in the proposal, but why should `schemaKey` take precendence?

Reply:
https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/forwarding-json-structured-logs.md#clusterlogforwarder-configuration

Reply:
@vimalk78 I am trying to understand the logic Schema starting with understanding `SchemaKey` ... let's say that I have 2 apps names `app1`, `app2`  I want `app1` json parsing to go to index `app1` and I want `app2` json parsing to go to index `app2`

I need to define two `schemaKey' values app1 and app2 right ???

 schemaKey: kubernetes.labels.app1
 schemaKey: kubernetes.labels.app2

Reply:
@vimalk78 so ... if I enable "json parsing" and there is a valid json for a log line ... are we going to send the message also to the regular index and not just to the "by schema" index ?

Reply:
s/"schema It takes"/"schema. It takes"

Reply:
No, you are sending the entire log record (which includes the `message` field) into a particular index. By using our Elasticsearch instance, this means into the "schema index".